pages/common/logstash.md

```diff
@@ -0,0 +1,12 @@
+# logstash
+
+> Migrate data from one source into another.
```

Comment:
I happen to think "migrate" provides a nice concise description of that process. The only downside is implying a move rather than a copy. Another problem is that "another" implies "another source", but in fact it supposedly means "destination" or "target". A slight rephrase may be in order for these reasons, but I'd love to keep the original proposal's conciseness when doing so.